### ISSUE TITLE
How to use predict on Ranking?

### ISSUE THREAD

**Issue by geegeet** [tensorflow/recommenders#248]
Hello, 

I'm quite new in the tensorflow world, and I'm having some issues with predicting using the basic Ranking model.
I followed [this](https://www.tensorflow.org/recommenders/examples/basic_ranking) tutorial, but unlike the retrieval tutorial, there's no prediction section (and it doesn't seem right to use BruteForce in a Ranking setting).

I searched in issues and found this https://github.com/tensorflow/recommenders/issues/215 but didn't succeed to make it work.

Is it possible to post a short example of making predictions in Ranking?

Thanks in advance,
Gee

**Comment by YannisPap** [comment]
I also struggled a little bit with this.

MovielensModel looks to me like a 'wrapper' around the RankingModel that defines a loss function for the training. The rating prediction takes place in the RaningModel.

e.g. in the compute_loss function we can see:
`rating_predictions = self.ranking_model(
        (features["user_id"], features["movie_title"]))`

@maciejkula, could you please confirm that we should use the RankingModel **not** the MovielensModel for inference on the specific example?

**Comment by dingrutao32** [comment]
I also want know how ranking models works

**Comment by windmaple** [comment]
This section should help:

https://www.tensorflow.org/recommenders/examples/basic_ranking#testing_the_ranking_model
